spec/features/auth_spec.rb

```diff
@@ -16,4 +16,11 @@
     expect(page).to have_content("user@example.com")
   end
 
+
+  scenario "Users can register" do
+    visit root_path
+    within(" .registation-form") { click_on "Register" }
+    save_and_open_page
```

Comment:
In general you won't want to leave `save_and_open_page` in your spec because it is a debugging tool